exercises/practice/largest-series-product/largest_series_product.cpp

```diff
@@ -0,0 +1,5 @@
+#include "largest_series_product.h"
+
+namespace largest_series_product {
+
+} // largest_series_product
```

Comment:
I'd suggest `// namespace largest_series_product` instead of `// largest_series_product`.